protostar/cli/network_command_util.py

```diff
@@ -75,10 +75,7 @@ def get_network_config(self) -> NetworkConfig:
 
     def get_gateway_client(self) -> GatewayClient:
         network_config = self.get_network_config()
```

Comment:
Is chain id needed in the network config anymore?
It definitely doesn't break anything so it could stay, but we should probably research it in the future.

Reply:
Yes, this value is still passed to the Signer class constructor.